doc/development/implementing_mandatory_sql_dialect_files.md

```diff
@@ -1,11 +1,11 @@
-# Implementing the SQL Dialect Adapter's Mandatory Classes 
+# Implementing the SQL Dialect Adapter's Mandatory Files 
 
 Each SQL Dialect adapter consists of two or more classes, depending on how standard-compliant the source behaves.
-The minimum implementation requires that you create the following mandatory classes:  
+The minimum implementation requires that you create the following mandatory files:  
   
 1. [`<Your_dialect_name>SqlDialect.java`](#creating-main-dialect-class)
-1. [`<Your_dialect_name>AdapterFactory.java`](#creating-adapter-factory)
-1. [`<Your_dialect_name>Adapter.java`](#creating-adapter)
+2. [`<Your_dialect_name>SqlDialectFactory.java`](#creating-the-sql-dialect-factory)
```

Comment:
Please, replace the numbers back to 1.
It's rendered to the correct list automatically. Otherwise we will need to change the numbers every time we remove or add something